devops/ansible_env

```diff
@@ -1,3 +1,4 @@
+export BUILD_NUM=${CIRCLE_BUILD_NUM}
```

Comment:
Ah, won't this not work if we try to run this locally? I get the following error when trying to spin this up:

```
TASK [Ensure CI environment vars are declared.] ********************************
fatal: [localhost]: FAILED! => {
    "assertion": "aws_ec2_ci_tags.build_num != \"\"", 
    "changed": false, 
    "evaluated_to": false, 
    "failed": true
}
```

(I don't hit this if I replace your line with `export BUILD_NUM=redshiftzero`)